### DISCUSSION TITLE
Why searching not working?

### DISCUSSION THREAD

**Discussion by ismatim** [shuding/nextra#436] [Q&A]
Hi all, a quick question, let me explain:

1. I cloned the nextra repository in my local.
2. Changed `index.mdx` file content with some dummy text.
3. Delete `.next` and rebuild all again.
4. I can't have any result by searching in the input box (top right).

What am I doing wrong?

ps: great library & great idea!

**Comment by ismatim** [comment]
Just clarifying my dumb question. Was checking the devtools it is requesting for:

* stork.wasm
* index-default.st

And, I had to grab the `stork.wasm` from `core` branch example because other `stork.wasm` file was not working for me.  Like : https://github.com/jameslittle230/stork/releases

**Reply by mi-hol** [reply]
> I had to grab the `stork.wasm` from `core` branch example because other `stork.wasm` file was not working for me. Like : https://github.com/jameslittle230/stork/releases

Would anybody know if this statement is still valid with stork V1.5 ?
